### DISCUSSION TITLE
How to rotate the canvas without input getting wildly out of sync?

### DISCUSSION THREAD

**Discussion by edthoms** [fabricjs/fabric.js#8130] [Q&A]
(Implementing freedrawing)

When the canvas is rotated using interact.js, putting the pen down anywhere has a strange behavior: the lines are drawn on a different place on the canvas. It seems as if there is an invisible "captor" that remains with the original dimensions and angle of translation, even though the canvas looks like it is being rotated. 

Any ideas of how to fix this behavior would be greatly appreciated, I don't think this is an issue with Fabric per se, but with the `<canvas></canvas>`

**Comment by ShaMan123** [comment]
Your use case is not clear enough to me
If you want to rotate the viewport use `setViewportTransform`
Some features are buggy with rotation I recall
